ui/src/entities/User/selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from 'react-redux';
+
+import { RootState } from 'src/duck';
+import { AuthState } from 'src/duck/auth';
+import { EmployeeState } from 'src/duck/employee';
+import { ProjectState } from 'src/duck/project';
+
+export const getUsersOrganizationId = () => {
```

Comment:
A user can have multiple organizations. I think we should rename this to `getActiveOrganization` or `getActiveUserOrganization`. What do you think?